packages/ng/step-by-step.md

```diff
@@ -0,0 +1,284 @@
+## Create the component in /src
+
+```
+$ ng g module lol -a=src
+installing module
+  create src\app\lol\lol.module.ts
+$ ng g component lol -a=src --export=true
+installing component
+  create src\app\lol\lol.component.scss
+  create src\app\lol\lol.component.html
+  create src\app\lol\lol.component.spec.ts
+  create src\app\lol\lol.component.ts
+  update src\app\lol\lol.module.ts
+```
+
+> - `-a=src` tells ng-cli to use application src, so it will generate everything in `/src/app/...`. by default it will use the demo application so you have to specify if you want to create something in `/src`

```

Comment:
i see what you mean, make src the default app

nope, i want the act of adding something to the lib be a deliberate act. i prefer people specify `ng g component lol -a=src` than people using `ng g` while trying to update the demo and pollute `/src` without realizing